python_bindings/src/PyExpr.cpp

```diff
@@ -22,7 +22,7 @@ void define_expr(py::module &m) {
             // PyBind11 searches in declared order,
             // int should be tried before float conversion
             .def(py::init<int>())
-            .def(py::init<float>())
+            //.def(py::init<float>())
```

Comment:
Please add a comment explaining why the line is commented out, or just delete it.